routes/auth.js

```diff
@@ -21,41 +21,8 @@ Cancer.find({}, function(err, cancers){
 
 // Renders account information, including the user's uploaded datasets
 exports.account = function(req, res){
-    User.findByGoogleId(req.session.passport.user)
-	.fail(function(err) {console.log(err);})
-	.then(function(user) {
-	    Dataset.datasetGroups({user_id: user._id}, function(err, groups){
-		// Throw error (if necessary)
-		if (err) throw new Error(err);
-		var user_id = String(user._id);
-		// here call to postgres for all annotations:
-		Aberrations.geneFind({aber_user_id: user_id}, 'right', function(err, geneAnnos) {
-		    if (err) throw new Error(err);
-		    // calculate the mode of each aberration
-		    geneAnnos = Aberrations.collateSourceAnnos(geneAnnos);
-
-		    Aberrations.geneFind({user_id: user_id}, 'left', function(err, sourceAnnos) {
-			if (err) throw new Error(err);
-			// calculate the mode of each aberration
-			sourceAnnos = Aberrations.collateSourceAnnos(sourceAnnos);
-
-			PPIs.ppiFind({user_id: user_id}, 'left', function (err, ppiAnnos) {
-			    if (err) throw new Error(err);
-			    // Render index page
-			    var pkg = { user: user,
-					 groups: groups,
-					 geneAnnos: geneAnnos,
-					 sourceAnnos: sourceAnnos,
-					 ppiAnnos: ppiAnnos,
-					 abbrToCancer: abbrToCancer,
-					 cancerToAbbr: cancerToAbbr,
-					 skip_requery: true};
-			    res.render('account', pkg);
-			});
-		    });
-		});
-	    });
-	});
+    var redirectURL = req.app.locals.annotationsURL + '/account/';
```

Comment:
The only actions we record now on the main MAGI site are uploading datasets.  I took the page down because it was requesting wrongly from the annotation site, so I'll bring it back.